install-rhel-deps-build-openjdk11.sh

```diff
@@ -117,6 +117,8 @@ TARBALL_VERSION="\${UPDATE}_\${BUILD}\${EA_SUFFIX}"
 PLATFORM_VERSION="\${PLATFORM}_\${TARBALL_VERSION}"
 TARBALL_NAME="\${TARBALL_BASE_NAME}-jdk_\${PLATFORM_VERSION}"
 TARBALL_NAME_JRE="\${TARBALL_BASE_NAME}-jre_\${PLATFORM_VERSION}"
+TARBALL_NAME_SHENDOH="\${TARBALL_BASE_NAME}-jdk-shenandoah_\${PLATFORM_VERSION}"
```

Comment:
Same `TARBALL_NAME_SHENDOH` comment. Either `TARBALL_NAME_SHGC` or `TARBALL_NAME_SHENANDOAH`

Reply:
Addressed the above comments and given a build in parallel. I will update again once the build is done.